src/CMakeLists.txt

```
@@ -19,13 +19,8 @@ string(REPLACE "-O3" "-O2" CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE}")
 add_library                     (leptonica ${src} ${hdr})
 set_target_properties           (leptonica PROPERTIES VERSION   6.0.0)
 set_target_properties           (leptonica PROPERTIES SOVERSION 6)
-if (WIN32)
-set_target_properties           (leptonica PROPERTIES OUTPUT_NAME       leptonica-${VERSION_PLAIN})
-set_target_properties           (leptonica PROPERTIES DEBUG_OUTPUT_NAME leptonica-${VERSION_PLAIN}d)
-else()
-set_target_properties           (leptonica PROPERTIES OUTPUT_NAME       leptonica)
-set_target_properties           (leptonica PROPERTIES DEBUG_OUTPUT_NAME leptonica)
-endif()
+
+set_target_properties           (leptonica PROPERTIES OUTPUT_NAME tesseract$<$<BOOL:${WIN32}>:-${VERSION_PLAIN}$<$<CONFIG:DEBUG>:d>>)
```

```suggestion
set_target_properties           (leptonica PROPERTIES OUTPUT_NAME leptonica$<$<BOOL:${WIN32}>:-${VERSION_PLAIN}$<$<CONFIG:DEBUG>:d>>)
```
Oups copy paste error